packages/react/src/__tests__/ReactContextValidator-test.js

```diff
@@ -323,13 +325,16 @@ describe('ReactContextValidator', () => {
 
     expect(() =>
       ReactTestUtils.renderIntoDocument(<ParentContextProvider />),
-    ).toWarnDev([
-      'Warning: MiddleMissingContext.childContextTypes is specified but there is no getChildContext() method on the ' +
-        'instance. You can either define getChildContext() on MiddleMissingContext or remove childContextTypes from ' +
-        'it.',
-      'Warning: Failed context type: The context `bar` is marked as required in `ChildContextConsumer`, but its ' +
-        'value is `undefined`.',
-    ]);
+    ).toWarnDev(
+      [
+        'Warning: MiddleMissingContext.childContextTypes is specified but there is no getChildContext() method on the ' +
+          'instance. You can either define getChildContext() on MiddleMissingContext or remove childContextTypes from ' +
+          'it.',
+        'Warning: Failed context type: The context `bar` is marked as required in `ChildContextConsumer`, but its ' +
+          'value is `undefined`.',
+      ],
+      {withoutStack: 1},
```

Comment:
Minor nit, since this only affects contributors and not users– I find this kind of confusing, both the naming and how it's overloaded.

We could perhaps address both of these issues– while still satisfying your goal of being highly visible– by changing splitting the public facing API into different methods:
* `toWarnDev`: Verifies warning and stack
* `toWarnDevWithoutStack`: Verifies warning and no-stack
* `toWarnDevWithMaxStackCount`: Verifies warning and maximum expected stack count

Just thinking out loud here. This change isn't necessary.

Reply:
This param was renamed while I was writing my original comment from `expectNoStack` to `withoutStack`. The new name is an improvement.